doc/source/ScriptingAPI/SceneContext/index.md

```diff
@@ -0,0 +1,25 @@
+# Scene Context #
+
+In the [Querying the Scene tutorial](../../Tutorials/Scripting/QueryingAScene/index.md), we demonstrated querying a scene using the `object()` method.
+
+Certain scene properties cannot be queried using the _Scene Inspector_ or the `object()` method. This is especially irksome, because the `object()` method was very convenient in that it bypassed the need to provide a [context](../Context/index.md). However, sometimes there is no way around this, and you will need to specify a context in which to query a scene property.
```

Comment:
"Certain scene properties cannot be queried using the _Scene Inspector_ or the `object()` method"

I think we do actually have convenience methods for all the scene properties.  It's not just object(), there's also bound() and transform() and attributes() and setNames().  Which doesn't help you come up with an example of where a user would actually want to use `with` ... hmm.

You could give an example of evaluating the scene at different frames using `with`?  Maybe accumulate the total bounding box of the scene for frames 1 - 24 or something?  Or you could stick with this example of accessing "object", but be clear that this is just illustrating what the accessors are doing behind the scenes, and that there is no real reason to do this.

Reply:
But I thought you mentioned there were some non-standard properties implemented by other DCCs that we don't have methods for?